system/settings/settings.xml

```diff
@@ -271,31 +271,33 @@
           </constraints>
           <control type="list" format="string" />
         </setting>
-        <setting id="musicplayer.replaygainpreamp" type="integer" parent="musicplayer.replaygaintype" label="641" help="36268">
+        <setting id="musicplayer.replaygainpreamp" type="number" parent="musicplayer.replaygaintype" label="641" help="36268">
           <level>3</level>
-          <default>89</default>
+          <default>0</default>
           <constraints>
-            <minimum>77</minimum>
-            <step>1</step>
-            <maximum>101</maximum>
+            <minimum>-15</minimum>
+            <step>0.2</step>
+            <maximum>15</maximum>
           </constraints>
-          <control type="spinner" format="string">
-            <formatlabel>14050</formatlabel>
+          <control type="slider" format="number">
+            <popup>true</popup>
```

Comment:
Do we use this for other settings? I definitely would prefer to see the slider "inline".